src/libm/Makefile

```diff
@@ -26,34 +26,44 @@ endif
 #
 
 ifeq ($(ARCH), x86_64)
-OBJ+=sleefdpsse2.o sleefdpavx.o
-OBJ+=sleefspsse2.o sleefspavx.o
+OBJ+=sleefdpsse2.o sleefdpavx.o sleefspsse2.o sleefspavx.o
+OBJ2+=sleefgdpsse2.o sleefgdpavx.o sleefgspsse2.o sleefgspavx.o
 
 ifeq ($(ENABLEAVX2), 1)
-OBJ+=sleefdpavx2.o
-OBJ+=sleefspavx2.o
+OBJ+=sleefdpavx2.o sleefspavx2.o
+OBJ2+=sleefgdpavx2.o sleefgspavx2.o
 endif
 
 ifdef ENABLEFMA4
-OBJ+=sleefdpfma4.o
-OBJ+=sleefspfma4.o
+OBJ+=sleefdpfma4.o sleefspfma4.o
 endif
 
 ifeq ($(ENABLEAVX512F), 1)
-OBJ+=sleefdpavx512f.o
-OBJ+=sleefspavx512f.o
+OBJ+=sleefdpavx512f.o sleefspavx512f.o
+OBJ2+=sleefgdpavx512f.o sleefgspavx512f.o
 endif
 
 endif
 
 #
 
 .PHONY: all
-all : libm.a sleef.h
+all : libm.a libgm.a sleef.h
 
 libm.a : $(OBJ)
 	$(FLOCK) libm.a.lock -c 'ar rcs libm.a $(OBJ)'
 
+libgm.a : $(OBJ2)
+	$(FLOCK) libgm.a.lock -c 'ar rcs libgm.a $(OBJ2)'
+
+OBJ.txt : $(OBJ)
```

Comment:
I see, thanks for the explanation. 

I am not aware of  a better way of doing this other than using`cmake` instead of using hand crafted Makefiles. Keep it as it is, we will think about moving to cmake in the future.